ddtrace/profiling/_asyncio.py

```diff
@@ -0,0 +1,61 @@
+import sys
+import typing
+
+
+try:
+    import asyncio
+except ImportError:
+    DefaultEventLoopPolicy = object
+
+    def get_event_loop_policy():
+        # type: (...) -> None
+        return None
+
+    def set_event_loop_policy(loop_policy):
+        # type: (...) -> None
+        pass
+
+
+else:
+    DefaultEventLoopPolicy = asyncio.DefaultEventLoopPolicy  # type: ignore[misc]
+
+    get_event_loop_policy = asyncio.get_event_loop_policy  # type: ignore[assignment]
+    set_event_loop_policy = asyncio.set_event_loop_policy  # type: ignore[assignment]
+
+    if hasattr(asyncio, "current_task"):
+        current_task = asyncio.current_task
+    else:
+
+        def current_task(loop=None):
+            return None
+
+    if hasattr(asyncio, "all_tasks"):
+        current_task = asyncio.all_tasks
```

Comment:
```suggestion
        all_tasks = asyncio.all_tasks
```
?

Also, is this because of these functions were introduced in Python 3.7? I think earlier Python versions have the same functions but in other places, e.g. `asyncio.Task.all_tasks` in Python 3.6.

Reply:
Good catch, I just fixed it. It's indeed for Python 3.7 and later only.  I'll check for Python 3.6